ui/components/multichain/connect-accounts-modal/__snapshots__/connect-accounts-modal.test.tsx.snap

```diff
@@ -354,17 +354,17 @@ exports[`Connect More Accounts Modal should render correctly 1`] = `
                       <div
                         class="mm-box currency-display-component mm-box--display-flex mm-box--flex-wrap-wrap mm-box--align-items-center"
                         data-testid="first-currency-display"
-                        title="$0.00 USD"
+                        title="0 ETH"
```

Comment:
I assume this snapshot was wrong for quite some time now.

The `providerConfig` on `mock-state.json` [here](https://github.com/MetaMask/metamask-extension/blob/develop/test/data/mock-state.json#L384-L390) is `0x5` (goerli), which is a testnet.

Test assets/accounts should not show any fiat values unless `showFiatInTestnets` is set to `true` (but it's currently `false` [here](https://github.com/MetaMask/metamask-extension/blob/develop/test/data/mock-state.json#L371))

You will see a bunch of snapshots changes in this PR, because of this ⬆️.